packages/types/src/interfaces/statement/definitions.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright 2017-2024 @polkadot/types authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+// order important in structs... :)
+/* eslint-disable sort-keys */
+
+import type { Definitions } from '../../types/index.js';
+
+import { runtime } from './runtime.js';
+
+export default {
+  rpc: {},
+  runtime,
+  types: {
+    SpStatementStoreStatementSource: {
+      _enum: ['Chain', 'Network', 'Local']
+    },
+    // SpStatementStoreStatement: {
```

Comment:
This type already exists in the substrate defs, so we can get rid of it. But that begs the questions should we be naming these types similar to the typegen? I would say no so there is no conflicts in the future.